.yarnrc.yml

```diff
@@ -2,6 +2,8 @@ nodeLinker: pnp
 
 npmRegistryServer: 'https://registry.npmjs.org/'
 
+npmAlwaysAuth: true
```

Comment:
This shouldn't be here 🤔

Reply:
It came from `master` :man_shrugging: https://github.com/smartcontractkit/external-adapters-js/blob/master/.yarnrc.yml#L5

I'll remove it